prefrencesHelper.js

```diff
@@ -1,9 +0,0 @@
-const workspace = require("vscode").workspace;
-
-function preferences() {
-    return workspace.getConfiguration(
-        "competitive-programming-helper"
-    );
-}
-
-module.exports = preferences;
```

Comment:
Why did you delete this file?

Reply:
There were two files `prefrencesHelper.js` and `preferencesHelper.js` which works the same. Also `prefrencesHelper.js` was misspelled and never used in codebase. Evidence showed it was an error so I removed it.

Reply:
Ah cool, thanks!